### PR TITLE
4.0 remove stray stuff

### PR DESCRIPTION
**Pull request by mariascharin**
@craigtaverner I took a stab at this (see my Slack message earlier today). Please review and let me know what you think. Thanks in advance!

### CODE REVIEW

File: cypher/cypher-docs/src/docs/dev/ql/schema-index/fulltext-schema-index.adoc [hunk 1]
```diff
@@ -43,13 +43,6 @@ While a composite index applies only to entities that match the indexed label an
 
 For information on how to configure full-text schema indexes, refer to <<operations-manual#index-configuration-fulltext-search,  Operations Manual -> Indexes to support full-text search>>.
 
-[NOTE]
```
Comment (craigtaverner) [inline cypher/cypher-docs/src/docs/dev/ql/schema-index/fulltext-schema-index.adoc:46]: I would think that users of 4.0 would still need to know how to port from 3.x to 4.0. Is there another place in the documentation that covers this? I think it is good to remove these bits and pieces here and there if there is a section of porting from 3.x, but not good to remove this if no such content exists.

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/CallTest.scala [hunk 2]
```diff
@@ -110,33 +104,33 @@ class CallTest extends DocumentingTest {
     }
 
     section("Call a procedure with literal arguments", "call-call-a-procedure-with-literal-arguments") {
-      p("""This calls the example procedure `org.neo4j.procedure.example.addNodeToIndex` using literal arguments.
+      p("""This calls the example procedure `dbms.security.createUser` using literal arguments.
```
Comment (craigtaverner) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/CallTest.scala:107]: The security procedures are deprecated in 4.0 (and in fact there was even a plan for complete removal, but it seems more likely we'll just deprecate). We should use some other examples.

Comment (craigtaverner) [reply cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/CallTest.scala:107]: We need an example that works on a normal database. Security procedures will only work on system databases (if they are not removed entirely). I favor having a user-built procedure, although a very small and simple one.

Comment (craigtaverner) [reply cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/CallTest.scala:107]: I have another idea. What about the `db.stats.*` procedures? They are new, work on normal databases (not system) and useful to normal users for support requests?

**Review by craigtaverner — CHANGES_REQUESTED** [review]

File: cypher/cypher-docs/src/docs/dev/ql/schema-index/fulltext-schema-index.adoc (hunk 1 above)
Comment (mariascharin) [reply cypher/cypher-docs/src/docs/dev/ql/schema-index/fulltext-schema-index.adoc:46]: That is a good point, and we should make sure there is a section about this in whatever "migrating to this new and shiny 4.0 release" document that will have to accompany the release. I have made a note of it in a card on our wall.

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/CallTest.scala (hunk 2 above)
Comment (mariascharin) [reply cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/CallTest.scala:107]: @craigtaverner Thank you! I will try it out.

Comment (mariascharin) [reply cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/CallTest.scala:107]: @craigtaverner As agreed in discussions yesterday, I will merge this to get the bug out, and raise a new card to find a different example.

**Review by davidoliverSP2 — APPROVED** [review]
Thanks @mariascharin 
I got this to build after rebasing.

**Pull request merged**